CHANGELOG.md

```diff
@@ -1,29 +1,14 @@
 # Lotus changelog
 
-# UNRELEASED
+# v1.28.0 / 2024-07-23
+This is the MANDATORY Lotus v1.28.0 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇.
 
-## ☢️ Upgrade Warnings ☢️
-
-- This Lotus release includes some correctness improvements to the events subsystem, impacting RPC APIs including `GetActorEventsRaw`, `SubscribeActorEventsRaw`, `eth_getLogs` and the `eth` filter APIs. Part of these improvements involve an events database migration that may take some time to complete on nodes with extensive event databases. See [filecoin-project/lotus#12080](https://github.com/filecoin-project/lotus/pull/12080) for details.
```

Comment:
Was it intentional to lose these warnings?

Reply:
Yes! These warnings were apart of the ##Unreleased section in the changelog. This warning is also present here in the Upgrade Warning section under the v1.28.0 Heading: https://github.com/filecoin-project/lotus/blob/a174046f0da0e062b37a7aeac2345f204424f8b0/CHANGELOG.md?plain=1#L13